static/src/javascripts/projects/commercial/modules/ad-verification/prepare-ad-verification.ts

```diff
@@ -41,35 +43,31 @@ const maybeRefreshBlockedSlotOnce: ConfiantCallback = (
 		},
 	);
 
-	// check if ad is blocked and haven't refreshed the slot yet.
-	if (
-		isBlocked &&
-		!!blockedSlotPath &&
-		!confiantRefreshedSlots.includes(blockedSlotPath)
-	) {
-		setForceSendMetrics(true);
-		captureCommercialMetrics();
+	// check if ad is blocked
+	if (isBlocked && !!blockedSlotPath) {
```

Comment:
Could we go a step further with the un-nesting and avoid the long if block?
```suggestion
	if (!isBlocked || !blockedSlotPath || confiantRefreshedSlots.includes(blockedSlotPath)) return
```
(pretty sure that logic is equivalent)

Reply:
Good shout. Actually the check for already refresh I’ve moved further down. It’s still worth capturing data if an ad is blocked twice, as this should not really happen.